Cargo.lock

```diff
@@ -88,11 +89,14 @@ dependencies = [
  "serde_json",
  "serde_urlencoded",
  "sha-1",
+ "sync_wrapper",
  "tokio",
- "tokio-tungstenite",
+ "tokio-tungstenite 0.15.0",
```

Comment:
We should update our other `tokio-tungstenite 0.14` dep to 0.15 as well. I'll do that after merging this.